usefulUtilities/modelJSONCleanup/modelJSONCleanup.js

```diff
@@ -0,0 +1,183 @@
+// Either supply the location of the config file in the argument or default to it being in the same folder as the script
+// Consts
+const CONFIG = require(process.argv[2] || './config.json');
+
+// Dependencies
+const fs = require('fs');
+const path = require('path');
+const modeljson = require(CONFIG.PATH_TO_MODELS_JSON);
```

Comment:
what about error handling? i.e. what if my config.json is bad? you need to validate the config file (i.e. make sure it contains all necessary key/value pairs) before continuing